.gitignore

```diff
@@ -22,3 +22,8 @@ _testmain.go
 *.exe
 *.test
 *.prof
+
+# Go dependency management files
+vendor
+Gopkg.toml
+Gopkg.lock
```

Comment:
Please delete this commit and this package will never vendor anything. So there is no need to add these to the gitignore. Thx!

Reply:
Ok!

Reply:
Done